SimG4Core/Application/src/RunManagerMTWorker.cc

```diff
@@ -131,15 +132,15 @@ struct RunManagerMTWorker::TLSData {
 
   TLSData() {}
 
-  ~TLSData() {}
+  ~TLSData() = default;
```

Comment:
please move to RunManagerMTWorker.h

Reply:
Likely no need to move to the header.

Reply:
@makortel @civanch you are correct, it was a mistake of mine during the review